pkg/infra/simulation/simulation.go

```diff
@@ -3,7 +3,7 @@ package simulation
 import (
 	"log"
 
-	baseagent "github.com/SOMAS2021/SOMAS2021/pkg/agents/default"
+	"github.com/SOMAS2021/SOMAS2021/pkg/agents"
 	agent1 "github.com/SOMAS2021/SOMAS2021/pkg/agents/team1/agent1"
 	agent2 "github.com/SOMAS2021/SOMAS2021/pkg/agents/team1/agent2"
```

Comment:
```suggestion
	"github.com/SOMAS2021/SOMAS2021/pkg/agents/team1/agent1"
	"github.com/SOMAS2021/SOMAS2021/pkg/agents/team1/agent2"
```

If you can please change the first lines of `agent1.go` and `agent2.go` to:
`package agent1` and `package agent2` respectively <3
Package should be called whatever the folder name is called

Reply:
(and it means you don't need to rename the import)